Makefile

```diff
@@ -6,6 +6,7 @@ IMAGE_TAG ?= $(shell git rev-parse --short HEAD)
 
 BUILD_DIR = out
 PACKAGES = $(shell go list ./...)
+LINT_PACKAGES = $(shell echo $(PACKAGES) | sed -e 's@github.com/fabric8-services/fabric8-jenkins-idler/internal/configuration@@')
```

Comment:
+1, could be improved further down the road if we have more than one package, file to exlude. For now it should do.